decentraland/unity-renderer
# fix: [BiW] The skybox is not set back correctly after teleporting

## What this PR includes?
- **BUG FIX:** "The skybox is not set back correctly after teleporting" (Fixes #311)
_After teleporting with /goto, staying in the BiW editor, the skybox is not set back correctly._

## How to test it?
1. Go to: https://play.decentraland.zone/?index.html&renderer=urn:decentraland:off-chain:renderer-artifacts:fix/BiW-The-skybox-is-not-set-back-correctly-after-teleporting&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100
2. Press `K` to open the Builder In World editor.
3. Open the chat window.
4. Type `/goto` + `{the coords you want}`.
5. Notice that the BiW editor is successfully closed and the sky back to have its normal texture.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
